features/diagnostics/DetailViewContent.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Diagnostics: Detail view content
+  Background:
+    Given the CSS selectors
+      | Alias   | Selector                               |
+      | details | [data-testid='code-block-diagnostics'] |
+
+  Scenario: Diagnostics detail view has expected content
+    Given the URL "/config" responds with
+      """
+      body:
+        mode: global
+      """
+
+    When I visit the "/diagnostics" URL
+    Then the page title contains "Diagnostics"
+    Then the "$details" element contains ""mode": "global""
+    Then the "$details" element contains ""environment": "universal""
```

Comment:
That's not always the case but I guess that's true for the fixtures that we have, right?

Reply:
That is true. Our fixture currently statically states environment to be universal. We could mock this out here, too, but it’s the same thing in the end.